docs/Antora/modules/howto_guides/pages/howto_measure_single_value.adoc

```diff
@@ -1,33 +1,374 @@
 = Measure a single value
 
-[#measure_single_value]
-== Measure a single value
+OpenDAQ offers a variety of solutions for reading Signal values, including Readers. 
 
-OpenDAQ offers a variety of solutions for reading signal values, including readers. For developers requiring access to only the most recent value, the signal interface provides a method named "getLastValue."
+For developers requiring access to only the most recent value, the Signal and Data Packet interfaces provide a method named `getLastValue` (`last_value` in Python).
+
+[#last_value_signal]
+== Getting the last value of a Signal
+
+When the `getLastValue` method (`last_value` in Python)  is called on a Signal, it yields an `IBaseObject` containing the last value.
+
+[tabs]
+====
+Cpp::
++
+[source,cpp]
+----
+// Get last value of a Signal
+auto lastValue = signal.getLastValue();
+----
+Python::
++
+[source,python]
+----
+# Get last value of a Signal
+last_value = signal.last_value
+----
+====
+
+[#last_value_data_packet]
+== Getting last value of a Data Packet
+
+Similar is true when calling the `getLastValue` method (`last_value` in Python)  on a Data Packet.
 
 [tabs]
 ====
 Cpp::
 +
 [source,cpp]
 ----
-void HowToDoc_MeasureSingleValue(daq::DevicePtr device)
-{
-    // get 1st signal of device...
-    auto signal = device.getSignalsRecursive()[0];
-    // ...and get a measurement value
-    auto measurementValue = signal.getLastValue();
-    std::cout << "Single measurement value: [" << measurementValue << "]" << std::endl;
-}
+// Get last value of a Data Packet
+auto lastValue = packet.getLastValue();
 ----
 Python::
 +
 [source,python]
 ----
-def HowToDoc_MeasureSingleValue(device):
-    signal = device.signals_recursive[0]
-    measurementValue = signal.last_value
-    print ("Single measurement value: [",measurementValue,"]")
+# Get last value of a Data Packet
+packet.last_value()  # TODO ()???
```

Comment:
Not sure if brackets are appropriate here... It's suppored to take one argument for Struct and the interpreter suggested it. Couldn't test Python code...